internal/events/dx_callbacks.go

```diff
@@ -218,6 +222,10 @@ func (em *eventManager) privateBlobReceived(dx dataexchange.Plugin, event dataex
 		event.Ack() // Still confirm the event
 		return
 	}
+	if br.Namespace != em.namespace {
+		log.L(em.ctx).Debugf("Ignoring blob from wrong namespace '%s'", br.Namespace)
```

Comment:
Thing a change from `wrong` to `different` might help users here, considering how many of these log entries they could see in their logs.

Comment applies to all the `wrong namespace` event entries, not just DX